api/previewCamera.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
Leftover. Was moved to https://github.com/PhotoboothProject/photobooth/pull/69.

Reply:
Those are shown here because both MR are to dev. As this branch is based on #69 all changes from that are in here, as well.